inductiva/resources/machines.py

```diff
@@ -84,17 +85,24 @@ def terminate(self):
             api_instance = inductiva.client.apis.tags.instance_api.InstanceApi(
                 client)
 
+            instance_group_config = \
+                inductiva.client.model.instance_group.InstanceGroup(
+                    name=self.name,
+                    machine_type=self.machine_type,
+                    num_instances=self.num_machines,
+                    spot=self.spot,
+                    disk_size_gb=self.disk_size_gb,
+                    zone=self.zone,
+                )
             try:
                 logging.info("Terminating machine group."
                              "This may take a few minutes.")
                 start_time = time.time()
-                api_instance.delete_instance_group(
-                    body=inductiva.client.model.instance.Instance(
-                        id=self.id, zone=self.zone))
-                termination_time_mins = (time.time() - start_time) / 60
+                api_instance.delete_instance_group(body=instance_group_config)
+                termination_time_mins = time.time() - start_time
                 logging.info(
-                    "Machine group of %s machines successfully"
-                    "terminated in %.2f mins.", self.num_machines,
+                    "Machine group of %s machines successfully "
+                    "terminated in %.2f s.", self.num_machines,
```

Comment:
You updated the string to indicate that the time is measured in seconds, but the variable is still `termination_time_mins`.